src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope_NES.c

```diff
@@ -262,9 +260,45 @@ void func_80821958(void* segment, u32 texIndex) {
     CmpDma_LoadFile(SEGMENT_ROM_START(item_name_static), texIndex, segment, 0x400);
```

Comment:
Can't see them, but with cmp doc'd perhaps we can name the above three functions. Seems like they load textures from an archive?

Reply:
Just a first attempt, but it's quite verbose:
`func_80821900` -> `Kaleido_LoadMapNameStaticSmallSegment`
`func_8082192C` -> `Kaleido_LoadMapNameStaticLargeSegment`
`func_80821958` -> `Kaleido_LoadItemNameStaticSmallSegment`

Reply:
Seems fine to me (perhaps remove `Small`? and only emphasize the large load, but can also see why to keep it).

Reply:
So perhaps
`func_80821900 -> Kaleido_LoadMapNameStaticSegment`
`func_8082192C -> Kaleido_LoadMapNameStaticSegmentLarge`
`func_80821958 -> Kaleido_LoadItemNameStaticSegment`

Could also drop `Segment` to align more with `CmpDma_LoadFile`, but then `Large` loses a bit of context?

Reply:
Went with dropping the `Segment` but could always add it back